code/game/objects/structures/crates_lockers/closets/secure/freezer.dm

```diff
@@ -12,6 +12,14 @@
 		else
 			icon_state = icon_opened
 
+/obj/structure/closet/secure_closet/freezer/close()
+	..()
+	if(!broken)
+		for(var/O in src)
+			if(istype(O,/obj/item/weapon/reagent_containers/food))
+				var/obj/item/weapon/reagent_containers/food/F = O
+				F.freshMod = -1
+

```

Comment:
No. It grows/lose temp to achieve 0 if not being affected, meaning that leaving it in the freezer is best. The moment it leaves, it begins to warm.